src/widget/range/range-widget.js

```diff
@@ -183,6 +183,7 @@ class RangeWidget extends Widget {
         props.min = Number( min );
         props.max = Number( max );
         props.step = Number( step );
+        props.value = this.element.value;
```

Comment:
I would call this `loadedValue` to avoid confusion since it's static.

Reply:
This makes sense. I iterated on this several times, trying to get a sense of the preferred conventions, but admittedly there's a fair bit to take in, so I decided that getting feedback like this would be more valuable for a first PR. 🙂